README.md

```diff
@@ -26,15 +26,22 @@ The server uses environment variables for tuning many of its parameters, as spec
 - `MODEL_FOLDER`: Folder from which to load the models. See [Loading Models](#Loading%20Models) below. Defaults to `./models`.
 - `MIN_CONFIDENCE`: Minimum score required for us to include a match in our results. Defaults to `0.8`.
 
-# Loading Models
+# Loading Object Detection Models
 
 On startup, the server looks in `MODEL_FOLDER` for any files named `<prefix>.frozen.pb`. It uses those prefixes to load those frozen model files into their own TensorFlow sessions, and looks for category mapping files named `<prefix>.label_map.pbtxt` and loads those as well, all in a model map indexed by prefix.
 
 When the server is shut down, an [`atexit`](https://docs.python.org/3/library/atexit.html) hook will `close()` all open TensorFlow sessions.
 
+# Loading Image classification Models
+
+On startup, the server looks in `MODEL_FOLDER` for any files named `<prefix>.pb`. It uses those prefixes to load those frozen model files into their own TensorFlow sessions, and looks for category mapping files named `<prefix>.pbtxt` and loads those as well, all in a model map indexed by prefix.
```

Comment:
We should standardize on our model/label suffixes. I'm fine not using `.frozen.pb` and `.label_map.pbtxt` and going with yours. Can you make that change in my script and in the docs here? Otherwise it's kind of odd, and also the `Models/README.md` is incorrect.

Reply:
Also, once you make that change, there's no reason to have these two separate "Loading" sections since they'll be identical.